src/behandlinger/behandlingsresultattyper.js

```diff
@@ -35,6 +35,14 @@ const behandlingsresultattyper = [
   {
     kode: 'MEDLEM_I_FOLKETRYGDEN',
     term: 'Medlem i folketrygden'
+  },
+  {
+    kode: 'AVSLUTTET_UTEN_ENDRING',
+    term: 'Behandlingen er avsluttet'
+  },
+  {
+    kode: 'HENLEGGELSE_BORTFALT',
```

Comment:
Det blir nok en migrering for denne. Vi har allerede henlagt saker som bortfalt hvor det nå står HENLEGGELSE

Reply:
Tok dette opp med fag, og det var akseptabelt for dem at vi starter med å endre på behandlingene "fra nå", ettersom disse behandlingsresultattypene ikke blir brukt i statistikk. Har likevel anbefalt migrering, og vi ble enige om det. Har laget script på det i API.